src/format.js

```diff
@@ -96,6 +97,17 @@ export function formatRelative(config, state, value, options = {}) {
     let defaults        = format && getNamedFormat(formats, 'relative', format);
     let filteredOptions = filterProps(options, RELATIVE_FORMAT_OPTIONS, defaults);
 
+    // Capture the current threshold values, then temporarily override them with
+    // specific values just for this render.
+    const thresholds = {...IntlRelativeFormat.thresholds};
+    Object.assign(IntlRelativeFormat.thresholds, {
+        second: 60, // seconds to minute
+        minute: 60, // minutes to hour
+        hour  : 24, // hours to day
+        day   : 30, // days to month
```

Comment:
The only question is what the days-to-month value should be. Should it be 28, 29, 30, or 31? With it set at `30`, by default it will go from `"29 days ago"` to `"1 month ago"`.